app/controllers/recipients_controller.rb

```diff
@@ -0,0 +1,41 @@
+class RecipientsController < ApplicationController
```

Comment:
Ok, yeah that makes sense in context.  I think the best thing to do here is to use semantic resources still, but namespace them for separation: http://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing.

This is trivial right now, and probably not necessary to perfect, as this is just a small learning app.  The custom routes will work fine for this.